docs/application/dotnet/guides/user-interface/nui/layouts.md

```diff
@@ -15,7 +17,15 @@ In addition, the layout framework allows nested layouts. `View` with a layout ca
 A `View` can be assigned to any layout and then can be re-assigned to a different layout. However, a layout can be used with one `View` at a time.
 
 New layouts can be implemented by deriving them from the layout base class and positioning the children according to the requirement. To create a custom layout, see [Creating custom Layouts](./custom-layout.md).
+## Common layouts
 
+| Type | Description | Appearance |
+| --- | --- | --- |
+| `AbsoluteLayout` | `AbsoluteLayout` is View's default layout that allowing to explicit positioning of chlid elements. The positions of children starts from the top left of the layout and cen be set by using the `Position` property.<br /><br /> [API Documentation](https://samsung.github.io/TizenFX/API9/api/Tizen.NUI.AbsoluteLayout.html) / [Guide](./absolute-layout.md) | ![AbsoluteLayout Example](./media/absolute-Layout.png) |
```

Comment:
`AbsoluteLayout` is the default `View` layout that allows to explicit position of child elements. The children's position starts from the top of the layout and can be set using the `Position` property.